plugin.js

```diff
@@ -25,6 +33,21 @@ function fastifyCookieSetCookie (reply, name, value, options, signer) {
   }
 
   const serialized = cookie.serialize(name, value, opts)
+
+  if (opts.enableWarnOnSafeLimit !== false) {
```

Comment:
so it is an opt-out? I would prefer an opt-in if we have excessive logging.

Reply:
It's opt-in now, changed it in the latest commit